sites/svelte-5-preview/src/routes/docs/content/03-appendix/03-deprecations.md

```diff
@@ -50,12 +50,11 @@ Svelte 5 introduces [event attributes](/docs/event-handlers) which deprecate eve
 <script>
 -	import { createEventDispatcher } from 'svelte';
 -	const dispatch = createEventDispatcher();
-+	let { onGreet } = $props();
++	let { greet } = $props();
 
-	function greet() {
+-	function greet() {
 -		dispatch('greet');
-+		onGreet?.();
```

Comment:
Shouldn't this be `greet` as well?
I have the feeling that this function either needs to be renamed, or can be removed from the example entirely (as greet seems to be given to the <button> directly now)

Reply:
This is getting removed as part of this PR. We're no longer suggesting switching to `onGreet`.

Reply:
Oops, I got confused by the double diff syntax 😅